<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>528</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Mesh_Computational_Geometry</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="plain_face_mode">
          <property name="text">
           <string>Plain Face</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="wireframe_mode">
          <property name="text">
           <string>Wireframe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="curvature_mode">
          <property name="text">
           <string>Curvature</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="voronoi_mode">
          <property name="text">
           <string>Vorono√Ø</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="show_points">
          <property name="text">
           <string>Show Points</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="show_triangle_connexions">
          <property name="text">
           <string>Show triangle connexions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="show_vertex_connexions">
          <property name="text">
           <string>Show vertex connexions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="show_laplacian">
          <property name="text">
           <string>Show Laplacian</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="show_normal">
          <property name="text">
           <string>Show Normal</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QComboBox" name="select_file_comboBox">
          <item>
           <property name="text">
            <string>Tetrahedron</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cube</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pyramide</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Queen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Slick</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="load_button">
          <property name="text">
           <string>Load File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_one_vertex">
          <property name="text">
           <string>Add 1 vertex (Step by step)</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_ten_vertices">
          <property name="text">
           <string>Add 10 vertices</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_all_vertices">
          <property name="text">
           <string>Add all vertices</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="exit_button">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="GLDisplayWidget" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLDisplayWidget</class>
   <extends>QWidget</extends>
   <header>gldisplaywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>exit_button</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
